### DISCUSSION TITLE
Does using AlDente increase battery temperatures?

### DISCUSSION THREAD

**Discussion by Paco089** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#310] [Q&A]
Hi,

I use a 16" MBP and since I have been using AlDente Pro I noticed slightly increased battery temperatures. I use my machine docked on a stand (lid open) with external monitor 90% of the time.

**Before using AlDente Pro:**

Battery temperature: 33-34C at idle (docked with external monitor + internal screen), fans inaudible

**With AlDente Pro:**

Battery temperature 35-36C at idle (docked with external monitor + internal screen), fan speed slightly increased

In fact since I have been using my MBP I have never seen it going above 35C (battery temperature) even under load and now it often sits at 36C at idle for hours.

**Should I be worried and are those operating temperatures (35-36C) still safe for the battery?**

**Comment by dmelgar** [comment]
As far as I can tell from looking at the code, Aldente shouldn't be able to raise the temperature of the battery. You should check "Activity Monitor" to see if you have a process using up lots of CPU. When sorted from highest to lowest you can see if something new or unexpected is showing up. CPU utilization is what causes the laptop to heat up, and therefore the battery and makes the fan spin up. The other key element is ambient temperature. Air temperature and also if its sitting on something warm like your lap.

**Reply by Paco089** [reply]
Yeah, it turned out that it was the ambient temperature and had nothing to do with AlDente. 

Thx for your reply!
